x-pack/plugins/apm/public/components/app/ErrorGroupDetails/Distribution/index.tsx

```diff
@@ -54,76 +43,72 @@ export function getFormattedBuckets(
-        heading={i18n.translate('xpack.apm.errorGroupDetails.noErrorsLabel', {
-          defaultMessage: 'No errors were found',
-        })}
-      />
-    );
-  }
-
-  const averageValue = mean(buckets.map((bucket) => bucket.y)) || 0;
   const xMin = d3.min(buckets, (d) => d.x0);
-  const xMax = d3.max(buckets, (d) => d.x);
-  const tickFormat = scaleUtc().domain([xMin, xMax]).tickFormat();
+  const xMax = d3.max(buckets, (d) => d.x0);
+
+  const xFormatter = niceTimeFormatter([xMin, xMax]);
+
+  const tooltipProps: SettingsSpec['tooltip'] = {
+    headerFormatter: (tooltip: TooltipValue) => {
+      const serie = buckets.find((bucket) => bucket.x0 === tooltip.value);
+      if (serie) {
+        return asRelativeDateTimeRange(serie.x0, serie.x);
+      }
+      return `${tooltip.value}`;
+    },
+  };
 
   return (
     <div>
       <EuiTitle size="xs">
         <span>{title}</span>
       </EuiTitle>
-      <Histogram
-        height={180}
-        noHits={distribution.noHits}
-        tooltipHeader={tooltipHeader}
-        verticalLineHover={(bucket: FormattedBucket) => bucket.x}
-        xType="time-utc"
-        formatX={(value: Date) => {
-          const time = value.getTime();
-          return tickFormat(new Date(time - getTimezoneOffsetInMs(time)));
-        }}
-        buckets={buckets}
-        bucketSize={distribution.bucketSize}
-        formatYShort={(value: number) =>
-          i18n.translate('xpack.apm.errorGroupDetails.occurrencesShortLabel', {
-            defaultMessage: '{occCount} occ.',
-            values: { occCount: value },
-          })
-        }
-        formatYLong={(value: number) =>
-          i18n.translate('xpack.apm.errorGroupDetails.occurrencesLongLabel', {
-            defaultMessage:
-              '{occCount} {occCount, plural, one {occurrence} other {occurrences}}',
-            values: { occCount: value },
-          })
-        }
-        legends={[
-          {
-            color: theme.euiColorVis1,
-            // 0a abbreviates large whole numbers with metric prefixes like: 1000 = 1k, 32000 = 32k, 1000000 = 1m
-            legendValue: numeral(averageValue).format('0a'),
-            title: i18n.translate('xpack.apm.errorGroupDetails.avgLabel', {
-              defaultMessage: 'Avg.',
-            }),
-            legendClickDisabled: true,
-          },
-        ]}
-      />
+      <div style={{ height: 180 }}>
+        <Chart>
+          <Settings
+            xDomain={{ min: xMin, max: xMax }}
+            tooltip={tooltipProps}
+            showLegend
+            showLegendExtra
+            legendPosition={Position.Bottom}
+          />
+          <Axis
+            id="x-axis"
+            position={Position.Bottom}
+            showOverlappingTicks
+            tickFormat={xFormatter}
+          />
+          <Axis
+            id="y-axis"
+            position={Position.Left}
+            ticks={2}
+            showGridLines
+            tickFormat={(value) => `${value} occ.`}
```

Comment:
Missing translation

Reply:
This is actually not needed. I removed it.
<img width="228" alt="Screenshot 2020-11-06 at 07 52 40" src="https://user-images.githubusercontent.com/55978943/98358195-17bb2c80-2005-11eb-8112-5bddc688ab08.png">

Reply:
Isn't it used on the axis? If not, why do we have it?

Reply:
ah, I see you removed it now. 👍